st2common/st2common/util/config_loader.py

```diff
@@ -254,6 +254,18 @@ def _get_datastore_value_for_expression(self, key, value, config_schema_item=Non
         if value:
             # Deserialize the value
             value = deserialize_key_value(value=value, secret=secret)
+            LOG.audit(
```

Comment:
I think this needs `if secret or "decrypt_kv" in value:`

Reply:
yep just pushed the new changes now